CMakeLists.txt

```diff
@@ -331,6 +336,9 @@ if(NOT OPTION_DEDICATED)
     link_package(Fontconfig TARGET Fontconfig::Fontconfig)
     link_package(Harfbuzz TARGET harfbuzz::harfbuzz)
     link_package(ICU_i18n)
+    link_package(Ogg TARGET Ogg::ogg)
+    link_package(Opus TARGET Opus::opus)
+    link_package(OpusFile TARGET OpusFile::opusfile)
```

Comment:
Only linking OpusFile should be needed. Opus and Ogg are deps of OpusFile, not direct deps of OpenTTD.